jobs/build-node-image.Jenkinsfile

```diff
@@ -74,9 +76,28 @@ lock(resource: "build-node-image") {
             }
         }
 
-        stage("Release Manifest") {
+        stage('Build Extensions Container') {
+            withCredentials([file(credentialsId: 'oscontainer-push-registry-secret', variable: 'REGISTRY_SECRET')]) {
+                // Use the node image as from
+                def build_from = container_registry_staging_manifest
+                // The Extesions container is using two repositories for now, we need to merge them to pass it as a secret
+                def repo_1 = "\$(pwd)/yumrepos/${stream_info.yumrepo.file}"
+                def repo_2 = "\$(pwd)/yumrepos/rhel-9.4.repo"
+                shwrap(""" cat ${repo_1} ${repo_2} > /tmp/final_repo """)
```

Comment:
I think instead of this, let's change `stream_info.yumrepo.file` to be e.g. `stream_info.yumrepo.files` instead (i.e. a list). Which we then concatenate if it's > 1.

That way the OCP release-specific repo hack and context live in the pipecfg, not directly in the pipeline.